docs/commands/build.md

```diff
@@ -20,6 +21,92 @@ this-is-param-case) in `dist/`
 
 ## Options:
 
-`--private`: Allows the building of private packages
+`--private`: Allows the building of private packages.
 
-`--preserve-modules`: Preserve module structure in generated modules
+`--preserve-modules`: Preserve module structure in generated modules.
+
+`--changed`: Build only packages whose workspaces contain files that have
+changed. Files that have changed are calculated comparing the current state of
+the repository with the branch specified by `compareBranch` or, if
+`compareBranch` is not set, with the default git branch.
+
+`--compareBranch`: Specify the comparison branch used to determine which files
+have changed when using the `changed` option. If this option is used without
+`changed`, the command will fail.
+
+`--descendants`: Build the packages specified by the `[packages...]` argument
+and/or the `--changed` option and additionally build all their descendants (i.e.
+the packages that have a direct or indirect dependency on them) in dependency
+order.
+
+`--ancestors`: Build the packages specified by the `[packages...]` argument
+and/or the `--changed` option and additionally build all their ancestors (i.e.
+the packages that have a direct or indirect dependency on them) in dependency
+order.
+
```

Comment:
In the descriptions for `--descendants` and `--ancestors`, what about the scenario where the user doesn't supply any `[packages...]` and doesn't use the `--changed` option?

Reply:
My assumption is that neither option will have any affect, because all packages are built. Assuming this is correct, might be useful to clarify that in these docs

Reply:
`modular build` standalone (unlike `modular test`) will just error because no package is specified. Not sure if it needs to be documented.